canonical-websites/insights.ubuntu.com
# Add news filter

## Done

Add news filter to posts

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Click on the filters and make sure there is a news filter, and that it returns only posts in the 'News' category


## Issue / Card

Fixes #220
